frontend/main-app/src/components/Login/Login.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import {Login} from "./Login";
+
+describe("Login component", () => {
+    it('loads and displays Login component', () => {
+        // Arrange
+        render(<Login/>);
+
+        // Act
+        const title = screen.getByRole("heading", {name: "Log in"});
+        const emailField = screen.getByLabelText("E-mail");
```

Comment:
For textfields, you can also try ```getByRole('textbox', { name: ..... });``` 😎

Reply:
Added where possible 😊 Interestingly, the password field does not have an ARIA-role by default, due to security concerns. [The testing library developers recommend using getByLabelText for passwords.](https://github.com/testing-library/dom-testing-library/issues/567)